doc/onboarding-technical.md

```diff
@@ -0,0 +1,82 @@
+## Who to CC in issues
+
+* `lib/child_process`: @cjihrig, @bnoordhuis, @piscisaereus
+* `lib/cluster`: @cjihrig, @bnoordhuis, @piscisaereus
+* `lib/stream*`: @nodejs/streams
+* `lib/net`: @indutny, @bnoordhuis, @piscisaereus, @chrisdickinson, @nodejs/streams
+* `lib/http`: @indutny, @bnoordhuis, @nodejs/http
+* `lib/vm`: @domenic
+* `lib/buffer`: @trevnorris
+* `src/async-wrap.*`: @trevnorris
+* `lib/{crypto,tls,https}`: @indutny, @shigeki, @nodejs/crypto
+* `src/node_crypto.*`: @indutny, @shigeki, @nodejs/crypto
+* `lib/timers`: @fishrock123, @misterdjules
+* upgrading v8: @bnoordhuis / @targos / @ofrobots
+* `lib/domains`: @misterdjules
+
+IN GENERAL: nodejs/ctc
+
+If you cannot find who to cc for a file, `git shortlog -n -s <file>` may help.
+
+
+## Labels
+
+### By Subsystem
+
+We generally sort issues generally sort by a concept of "subsystem" so that we know what part(s) of the codebase it touches.
+
+**Subsystems generally are**:
+
+* `lib/*.js`
+* `doc`, `build`, `tools`, `test`, `deps`, `lib / src` (special), and there may be others.
+
+### Other Labels
+
+* OS labels
+* `semver-{minor,major}`
+  * be conservative – that is, if a change has the remote *chance* of breaking something, go for semver-major
+  * when adding a semver label, add a comment explaining why you're adding it
+  * minor vs. patch: roughly: "does it add a new method / does it add a new section to the docs"
+  * major vs. everything else: run last versions tests against this version, if they pass, **probably** minor or patch
+  * A breaking change helper ([full source](https://gist.github.com/chrisdickinson/ba532fa0e4e243fb7b44)):
+  ```
+  git checkout $(git show -s --pretty='%T' $(git show-ref -d $(git describe --abbrev=0) | tail -n1 | awk '{print $1}')) -- test; make test
+  ```
```

Comment:
Perhaps add the LTS labels to this? Or perhaps we need a separate document discussion LTS management?